ChangeLog.md

```diff
@@ -11,6 +11,39 @@ GitHub release pages and in the git history.
 
 ## \[Unreleased\]
 
+### Main changes
+
+In Python API, `spglib.get_spacegroup_type` and `spglib.get_spacegroup_type_from_symmetry` now return a `spglib.SpaceGroupType` object, inheriting `dataclasses.dataclass`, instead of a plain `dict`.
+The `dict`-like interface is deprecated and will be removed in the future.
+
+For backward compatibility, the `dict`-like interface is still available:
+
+```python
+spgtype: SpaceGroupType | None = get_spacegroup_type(cell)
+assert spgtype is not None
+print(spgtype['number'])
+print(list(spgtype.items()))
+```
+
+However, we recommend using the `dataclass` interface:
+
+```python
+spgtype: SpaceGroupType | None = get_spacegroup_type(cell)
+assert spgtype is not None
+print(spgtype.number)
+
+# If you really need dictionary, ...
+from dataclasses import asdict
+print(asdict(spgtype))
+```
+
+We consider migrating plain `dict` into `dataclasses.dataclass` for the returned objects of other Python APIs.
```

Comment:
I think we can address this within the `2.5.0` release

Reply:
OK. I'll remove this line.